protect_archiver/client/legacy.py

```diff
@@ -14,16 +16,16 @@ def __init__(
         username: str,
         password: str,
         verify_ssl: bool,
-    ):
+    ) -> None:
         self.protocol = protocol
         self.address = address
         self.port = port
         self.username = username
         self.password = password
         self.verify_ssl = verify_ssl
 
-        self._access_key = None
-        self._api_token = None
+        self._access_key: Optional[str] = None
+        self._api_token: Optional[str] = None
```

Comment:
Same as in the other location; what's your idea behind marking these as optional? Maybe I'm missing something...

Reply:
@danielfernau same explanation as above, we clearly see here it defaults to None if even though we might expect a string type hence `Optional[str]`

Reply:
@ch3ck That makes sense, thanks for the explanation!